app/models/house_member.rb

```diff
@@ -0,0 +1,3 @@
+class HouseMember < ApplicationRecord
+
```

Comment:
May want to have a many_through for later extensions connecting single house members to multiple users (user approval rating of that member?)